app/uk/gov/hmrc/cipphonenumberstubs/services/GovNotifyStubService.scala

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cipphonenumberstubs.services
+
+import play.api.Logging
+import play.api.libs.json.Json
+import play.api.mvc.Result
+import play.api.mvc.Results.{Created, Ok}
+import uk.gov.hmrc.cipphonenumberstubs.services.responses.{NotificationResponse, VerificationResponses}
+import javax.inject.{Inject, Singleton}
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton()
+class GovNotifyStubService @Inject() (implicit val executionContext: ExecutionContext) extends Logging {
+
+  def sms = Future {
+    Created(Json.parse(VerificationResponses.notificationResponse))
+  }
+
+  def status(notificationId: String): Future[Result] = Future {
+      Ok(Json.parse(parseNotificationResponse(notificationId)))
+  }
+
+  def parseNotificationResponse(notificationId: String) = {
+    notificationId match {
+      case "16770ea0-d385-4b17-a0b4-23a85c0c5b1a" => NotificationResponse.sending
```

Comment:
i believe there are other scenarios that need adding here:

        case "created" => (101, "Message is in the process of being sent")
        case "sending" => (102, "Message has been sent")
        case "pending" => (103, "Message is in the process of being delivered")
        case "sent" => (104, "Message was sent successfully")
        case "delivered" => (105, "Message was delivered successfully")
        case "permanent-failure" => (106, "Message was unable to be delivered by the network provider")
        case "temporary-failure" => (107, "Message was unable to be delivered by the network provider")
        case "technical-failure" => (108, "There is a problem with the notification vendor")